src/config.js

```diff
@@ -16,5 +16,6 @@ export default {
     title: null,
     body: null
   },
-  oidcEnabled: process.env.VUE_APP_OIDC_ENABLED === 'true'
+  oidcEnabled: process.env.VUE_APP_OIDC_ENABLED === 'true',
+  showsFeedbackButton: true
```

Comment:
From reading the PR description, I think this should be `false` by default. Then it could be set to `true` on whatever servers should show the button.

Reply:
Yes, agreed